packages/react-frontend/src/DiaryEntry.jsx

```diff
@@ -105,6 +106,7 @@ function DiaryEntry() {
 
   return (
     <div className="diary-container">
+      <HamburgerMenu />
```

Comment:
To change

Reply:
Thanks for pointing this out. I didn't even check all the different pages to make sure it stayed in the top corner. I will fix this issue now.